XiaoMi/rdsn
# refactor: remove simple_load_balancer

remove simple_load_balancer, and make the interface server_load_balancer simple.

### Manual Test
1. start onebox with 5 replica servers and 3 meta server2
2. create a table name with "temp", we can see create table process is ok.
3. kill 3 replica servers
4. use ddd_diagnose to see the ddd partitions
```>>> ddd_diagnose
Total 1 ddd partitions:

(1) 2.3
    config: ballot(6), last_committed(8)
    ----
    dropped[0]: node(10.231.57.100:34801), drop_time(2021-09-01 18:56:19.670), alive(false), collected(false), ballot(-1), last_committed(-1), last_prepared(-1)
    dropped[1]: node(10.231.57.100:34802), drop_time(2021-09-01 18:56:29.673), alive(false), collected(false), ballot(-1), last_committed(-1), last_prepared(-1)  <== the secondary latest
    dropped[2]: node(10.231.57.100:34803), drop_time(2021-09-01 18:56:43.560), alive(false), collected(false), ballot(-1), last_committed(-1), last_prepared(-1)  <== the latest
    ----
    last_drops[0]: node(10.231.57.100:34801)
    last_drops[1]: node(10.231.57.100:34802)  <== the secondary latest
    last_drops[2]: node(10.231.57.100:34803)  <== the latest
    ----
    ddd_reason: the last dropped node(10.231.57.100:34803) haven't come back yet

Proposed count: 0/1

Diagnose ddd done.
```
